src/HttpException.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
This will change their namespace too. Also I don't see any need to put exceptions to one folder.

We should not apply questionable decisions from the old connect.